src/lib/constants.ts

```
@@ -99,6 +99,6 @@ export const MAKE_TOAST_MESSAGES = {
 /** The URL for the KCL samples manifest files */
 export const KCL_SAMPLES_MANIFEST_URLS = {
   remote:
-    'https://raw.githubusercontent.com/KittyCAD/kcl-samples/main/manifst.json',
+    'https://raw.githubusercontent.com/KittyCAD/kcl-samples/main/manifest.json',
```

Looks like the fallback was completely masking the fact that this URL was bad.